adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java

```diff
@@ -2520,4 +2529,8 @@ void countDownLatch() {
             }
         }
     }
+
+    private Context getContext() {
+        return InstrumentationRegistry.getContext();
```

Comment:
why do you need this wrapper method, specially since you are implementing in more than one class